### DISCUSSION TITLE
Command argument sent to qBittorrent

### DISCUSSION THREAD

**Discussion by say-no-to-deer-pressure** [cross-seed/cross-seed#219] [Q&A] [answered]
Hi, thank you for your amazing software! 
Unfortunately I'm having an issue getting qbit to pass correct information to cross-seed to search for the torrent (and then hopefully inject it into qbit afterwards)
I have both cross-seed and qbit running from Docker containers through Unraid. cross-seed is running in daemon mode. I have modified config.js as needed (I put it [here ](https://pastebin.com/hnSQRZ91)if you want to verify the settings. )

In Tools > Options > Downloads, I checked "Run external program on torrent completion" and entered this:

curl -XPOST http://192.168.1.171:2468/api/webhook --data-urlencode 'name="%N"'

I checked the cross-seed logs and this was the output:

> 
> [32m[32minfo[32m[39m: [server] Received search request: {}
> [32m[32minfo[32m[39m: [server] Found 0 torrents for {}

There is nothing between the curly braces, so I'm assuming the torrent name isn't being passed correctly. I tried passing  %N both with and without double quotations, but no luck.
Could you kindly tell me what I'm doing wrong?

Thank you.

**Comment by mmgoodnow** [comment]
I think you can remove the double quotes

**Reply by say-no-to-deer-pressure** [reply]
Thanks for the quick reply. Taking away the quotes gives the following error:

```
[31m[31merror[31m[39m: no working indexers available
[32m[32minfo[32m[39m: Validating your configuration...
[32m[32minfo[32m[39m: Your configuration is valid!
[32m[32minfo[32m[39m: [server] Server is running on port 2468, ^C to stop.
[31m[31merror[31m[39m: [server] A name or info hash must be provided
[32m[32minfo[32m[39m: [server] Received search request: {}
[32m[32minfo[32m[39m: [server] Found 0 torrents for {}
```

And just for clarity, this is what I'm passing via qBittorrent:

`curl -XPOST http://192.168.1.171:2468/api/webhook --data-urlencode 'name=%N'`

**Reply by say-no-to-deer-pressure** [reply, marked as answer]
I figured it out! The argument is only passed correctly if you use double quotes. This works for me:

curl -XPOST http://192.168.1.171:2468/api/webhook --data-urlencode "name=%N"

Might want to make a note of that somewhere in your documentation :)
